dm/master/server.go

```diff
@@ -182,10 +184,12 @@ func (s *Server) Start(ctx context.Context) (err error) {
 	// And curl or safari browser does trigger this problem.
 	// But I haven't figured it out.
 	// (maybe more requests are sent from chrome or its extensions).
+	s.InitOpenAPIHandles()
```

Comment:
What about this format?
```suggestion
	s.echo, err = InitOpenAPIHandles()
```

Reply:
i think `func (s *Server) InitOpenAPIHandles() error` is better, because in L68, i need to call(openapi.RegisterHandlers(e, s))